geonode/security/models.py

```diff
@@ -396,6 +399,10 @@ def get_user_perms(self, user):
         """
         Returns a list of permissions a user has on a given resource
         """
+        # To avoid circular import
+        from geonode.base.models import Configuration
+
+        config = Configuration.load()
         ctype = ContentType.objects.get_for_model(self)
         PERMISSIONS_TO_FETCH = VIEW_PERMISSIONS + ADMIN_PERMISSIONS + LAYER_ADMIN_PERMISSIONS
```

Comment:
@marthamareal I would also move VIEW_PERMISSIONS, ADMIN_PERMISSIONS, LAYER_ADMIN_PERMISSIONS into the permissions module.

Reply:
Done